backend/app/views/spree/admin/shared/_store_switcher.html.erb

```diff
@@ -12,5 +12,11 @@
       <% @stores.each do |store| %>
         <%= store_switcher_link(store) %>
       <% end %>
+
+      <div class="dropdown-divider m-0"></div>
```

Comment:
@MatthewKennedy we need to wrap it in a `if can? :create, Spree::Store` check

Reply:
Sorted.